labsTemplate/README.md

```diff
@@ -0,0 +1,98 @@
+# auroLabs - [Name]
+
+`<[namespace]-[name]>` is an experimental [HTML custom element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) only available as a PRE RELEASE. Users of this element understand the risks of using a pre-released version of this element.
```

Comment:
I would say users of this element "accept" the risk. We don't know that they actually understand it but we can certainly assert that using the component is acceptance of all the associated risks.

Reply:
Could be good to add a question followup to the generator script if they say it's an AuroLabs component to followup with something to the affect of....

"Do you understand the risks of using an AuroLabs component in a production environment?"

Reply:
I don't think we need to go that far. Fact is, the devs know what they are doing and submit code to production apps all the time. The risk here is that it's a pre-release and there is the risk of radical changes to the CE itself. But I feel that is clearly communicated in the term `pre-release`.